src/universal-dapp.js

```diff
@@ -549,11 +549,14 @@ UniversalDApp.prototype.getCallButton = function (args) {
 
     var decoded
     self.runTx({ to: args.address, data: data, useCall: args.abi.constant && !isConstructor }, function (err, txResult) {
-      var result = txResult.result
+      var result
       if (err) {
         replaceOutput($result, $('<span/>').text(err).addClass('error'))
       // VM only
-      } else if (self.executionContext.isVM() && result.vm.exception === 0 && result.vm.exceptionError) {
```

Comment:
This seems to be wrong, all the conditions below are error cases.

Maybe it would be better adding an extra condition before all these:
```
if (!err) {
  result = txResult.result
}
```